### ISSUE TITLE
Themes syntax error

### ISSUE THREAD

**Issue by monoflash** [primefaces/primeui#241]
PrimeUI 4.1.8 found css syntax error:

```
gulp-postcss: primeui/themes/aristo/theme.css
autoprefixer: primeui/themes/aristo/theme.css:67:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:85:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:108:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:132:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:160:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:175:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:1108:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:1133:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:1182:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:1207:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/aristo/theme.css:1234:5: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.

gulp-postcss: primeui/themes/bootstrap/theme.css
autoprefixer: primeui/themes/bootstrap/theme.css:37:3: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/bootstrap/theme.css:555:3: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/bootstrap/theme.css:595:3: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/bootstrap/theme.css:619:3: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/bootstrap/theme.css:698:3: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.

gulp-postcss: primeui/themes/delta/theme.css
autoprefixer: primeui/themes/delta/theme.css:32:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:386:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:407:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:438:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:467:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:514:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:531:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:550:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
autoprefixer: primeui/themes/delta/theme.css:565:2: Gradient has outdated direction syntax. New syntax is like `to left` instead of `right`.
```

**Comment by cagataycivici** [comment]
Could you please provide a pull request?

**Comment by cagataycivici** [comment]
This was fixed in 4.1.9.
